tools/log_processing/ems_refresh_timings.rb

```diff
@@ -0,0 +1,155 @@
+RAILS_ROOT = ENV['RAILS_ENV'] ? Rails.root : File.expand_path(File.join(__dir__, %w(.. ..)))
+$LOAD_PATH.push File.join(RAILS_ROOT, 'gems/pending/util') unless ENV['RAILS_ENV']
+
+require 'miq_logger_processor'
+require 'active_support/core_ext/enumerable' # Pull in Enumerable sum method
+require 'more_core_extensions/core_ext/array'
+require 'trollop'
+require 'time'
+
+def parse_args(argv)
+  options  = {}
+  logfiles = []
+
+  opts = Trollop.options do
+    banner <<-EOS
+Parse EMS Refreshes from a set of evm.log files and filter on a set of
+provided conditions.
+
+Usage:
+  ruby ems_refresh_timings.rb [OPTION]... <FILE>...
+
+Options:
+EOS
+    opt :sort_by, 'Column to sort by, options are start_time, end_time, '\
+                  'total_time, ems, target_type, and target',
+        :type => :string, :default => 'start_time'
+    opt :target, 'Filter by specific targets, comma separated', :type => :string
```

Comment:
If you specify the type here as `:type = :strings`, then trollop will split on comma-separated for you (same with the options below)